src/components/v2/devtronStackManager/AboutDevtronView.tsx

```diff
@@ -0,0 +1,130 @@
+                                exact
+                                to={tab.link}
+                                activeClassName="active"
+                                className="tab-list__tab-link no-decor"
+                            >
+                                {tab.name}
+                            </NavLink>
+                            {selectedTabIndex == index && <div className="about-devtron__active-tab" />}
+                        </li>
+                    )
+                })}
+            </ul>
+        )
+    }
+
+    const renderTabData = (): JSX.Element => {
+        if (selectedTabIndex === 0) {
+            return (
+                <p className="about-devtron__view-info fs-13 fw-4 m-0">
+                    A web based CI/CD Orchestrator leveraging Open Source tools to provide a No-Code, SaaS-like
+                    experience for Kubernetes
+                </p>
+            )
+        }
+
+        return (
+            releaseNotes && (
+                <div className="about-devtron__view-releases">
+                    {releaseNotes.map((releaseNote, idx) => {
+                        return (
+                            <div key={`release-notes-${idx}`} className="about-devtron__release-notes-wrapper cn-9">
+                                <h2 className="about-devtron__release-name fs-16 fw-6">
+                                    Release {releaseNote.releaseName}
+                                </h2>
+                                <MarkDown
+                                    className="about-devtron__release-notes fs-14 fw-4"
+                                    breaks={true}
+                                    markdown={
+                                        releaseNote.body.startsWith('##')
+                                            ? releaseNote.body.replace('##', '')
+                                            : releaseNote.body
+                                    }
+                                />
+                            </div>
+                        )
+                    })}
+                </div>
+            )
+        )
+    }
+
+    return (
+        <div className="about-devtron__view-container flex column left top">
+            <img className="about-devtron__view-image" src={AboutDevtron} alt="About Devtron" />
+            <div className="about-devtron__view-wrapper">
+                <div className="about-devtron__details">
+                    <h2 className="about-devtron__view-heading cn-9 fs-20 fw-6">
+                        Devtron {serverInfo?.currentVersion ? `(${serverInfo.currentVersion.toLowerCase()})` : ''}
+                    </h2>
+                    <div className="about-devtron__view-tabs w-100">
+                        {renderTabs()}
+                        {renderTabData()}
+                    </div>
+                </div>
+                <InstallationWrapper
+                    installationStatus={serverInfo?.status}
```

Comment:
can we remove this installationStatus? as we are sending whole serverInfo to the component. We can take it from there on the child

Reply:
We're using `InstallationWrapper` for both server & module installation so the installation status will be fetched from different objects i.e. for server installation, it'll be fetched from serverInfo where as for module it'll be ref'ed from ModuleDetails.